README.md

```diff
@@ -25,7 +25,7 @@ Run Tests
 ----
 
 ```
-$ npm test
+$ NODE_ENV=local npm test
```

Comment:
@yorkie hi, i have remove in this commit https://github.com/weflex/semver-increment/pull/7/commits/b7bda5203d92f5d22960bf61969d060927509cb1 , also add `NODE_ENV` in `.travis.yml`